packages/react/src/Heading/Heading.module.css

```diff
@@ -144,3 +144,97 @@
     font-weight: var(--base-text-weight-light);
   }
 }
+
+/**
+ * Heading widths
+ */
+
+.Heading--width-condensed,
+.Heading-narrow--width-condensed {
+  font-variation-settings: 'wdth' 75;
```

Comment:
`font-stretch` is also supported as an attribute, and accomplishes the same thing 👀 maybe move to that to allow `font-variation-settings` to be used for other settings (and reduce risk of conflicting utilities)? The value just need to be set in `%` ✨

Reply:
Thanks @tobiasahlin! I've switched to `font-stretch` 👍